src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ToModelTransformers.kt

```diff
@@ -446,6 +460,129 @@ fun toHdc(
   submittedByFullName = licence.getSubmittedByFullName(),
 )
 
+fun transformToSarLicence(licence: ModelLicence) = SarLicence(
```

Comment:
This class is massive - could we move this SAR specific to a separate class in `src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/publicApi/model/subjectAccessRequest`